cmd/cli/kubectl-kyverno/utils/common/common.go

```diff
@@ -510,21 +509,18 @@ OuterLoop:
 
 	policyContext = policyContext.WithNewResource(mutateResponse.PatchedResource)
 
-	var info Info
 	var validateResponse engineapi.EngineResponse
 	if policyHasValidate {
 		validateResponse = eng.Validate(context.Background(), policyContext)
```

Comment:
I believe we should make use of `*ResultCounts` in the validate function as `ProcessValidateEngineResponse` is removed and it updates the values of `ResultConuts`

When I tried to apply a kyverno policy on a resource without generating a policy report, I got zero value for all Pass, Fail, Warn, Error and Skip.
```
[mariamfahmy@mariamfahmy kyverno]$ ./cmd/cli/kubectl-kyverno/kubectl-kyverno apply test/cli/apply/policies/policy.yaml --resource test/cli/apply/resource/resources.yaml 

Applying 2 policy rules to 2 resources...

pass: 0, fail: 0, warn: 0, error: 0, skip: 0 ```

Reply:
I made some changes 🤞

Reply:
@MariamFahmy98 counts should be good now, let me know if it is not the case.